components/PlantList.js

```diff
@@ -2,19 +2,21 @@ import styled from "styled-components";
 import PlantCard from "./PlantCard";
 
 export default function PlantList({ plants, toggleFavourite }) {
-  if (!plants || plants.length === 0) return <p>No plants found</p>;
+  if (!plants || plants.length === 0)
+    return (
+      <p>
```

Comment:
Consider styling the message that displays when there are no plants. Or do you already defined this in the global styling?

Reply:
the p-element itself is not defined. but we styled the card and the color of the text in styles.js